build.gradle

```
@@ -239,6 +266,26 @@ dependencyManagement {
   }
 }
 
+project.ext {
+  pactVersion = getCheckedOutGitCommitHash()
+}
+
+def static getCheckedOutGitCommitHash() {
+  'git rev-parse --verify --short HEAD'.execute().text.trim()
+}
+
+task runProviderPactVerification(type:Test) {
+  logger.lifecycle("Runs provider pact Tests")
+  useJUnitPlatform()
+  testClassesDirs = sourceSets.contractTest.output.classesDirs
+  classpath = sourceSets.contractTest.runtimeClasspath
+  systemProperty 'pact.verifier.publishResults', System.getProperty('pact.verifier.publishResults')
+  systemProperty 'pact.provider.version', project.pactVersion
+  include "uk/gov/hmcts/ccd/v2/external/controller/**"
```

```suggestion
  include "uk/gov/hmcts/reform/hmc/controllers/**"
```
Use path in this project